scripts/linux/nestedAgentLock.sh

```diff
@@ -146,21 +146,23 @@ function update_capabilities() {
 --request PUT "https://msazure.visualstudio.com/_apis/distributedtask/pools/$POOL_ID/agents/$agentId/usercapabilities" \
 -H "Content-Type:application/json" \
 -H "Accept: application/json;api-version=5.0;" \
---max-time 10 \
---retry 5 \
+--max-time 15 \
+--retry 10 \
 --retry-delay 0 \
---retry-max-time 40 \
+--retry-max-time 80 \
 --retry-connrefused \
 --data @<(cat <<EOF
 $newAgentUserCapabilities
 EOF
 ))
+
     # Validate the capability update was successful
     responseUserCapabilities=$(echo $responseCapabilities | jq '.userCapabilities')
 
     if [ "$responseUserCapabilities" != "$newAgentUserCapabilities" ]
     then
-        echo "Capabilities were not updated properly. This will be retried."
+        echo "Capabilities were not updated properly. Dumping response below." >&2
+        echo "$responseCapabilities" | jq '.' >&2
```

Comment:
What happens if the $responseCapabilities are not parsable by JSON ? In that case we would get a generic parse error right and not the actual response since your are piping it into jq ?

Reply:
Ah that's a good point. I was assuming the response would still be json but I just tested this logic path and devops isn't returning json (tested by mangling the url resource path so devops returns some error).  Fixed to remove the jquery pipe and it will log the error now.